prime-dotnet-webapi/Models/SelfDeclaration.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Newtonsoft.Json;
+
+namespace Prime.Models
+{
+    [Table("SelfDeclaration")]
+    public class SelfDeclaration : BaseAuditable, IEnrolleeNavigationProperty
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [Required]
```

Comment:
[Required] just means not null, so this doesn't make any difference on a field of type int.